### ISSUE TITLE
0.26.5 Breaks `async with` for `asyncpg.create_pool`

### ISSUE THREAD

**Issue by Bluesy1** [bryanforbes/asyncpg-stubs#121]
Commit e387d77f6af463e8cebe3e4601ed44adfb9cc6d0 has pyright raising the following type errors:
```
Error: Object of type "Coroutine[Any, Any, Pool[Record]]" cannot be used with "with" because it does not implement __aenter__ (reportGeneralTypeIssues)
Error: Object of type "Coroutine[Any, Any, Pool[Record]]" cannot be used with "with" because it does not implement __aexit__ (reportGeneralTypeIssues)
```
This directly goes against  what the docs for asyncpg say: https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg.pool.create_pool

**Comment by bryanforbes** [comment]
I was so focused on fixing the return type of `create_pool()` for one case, that I broke another. I'll fix this ASAP.

**Comment by bryanforbes** [comment]
This issue is resolved and is included in 0.26.6 which has been published to pypi.

**Comment by Bluesy1** [comment]
Thanks for the quick fix!
